sdk/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration/_app_configuration_management_client.py

```diff
@@ -57,7 +59,7 @@ class AppConfigurationManagementClient:  # pylint: disable=client-accepts-api-ve
     :type credential: ~azure.core.credentials.TokenCredential
     :param subscription_id: The Microsoft Azure subscription ID. Required.
     :type subscription_id: str
-    :param base_url: Service URL. Default value is "https://management.azure.com".
+    :param base_url: Service URL. Default value is empty.
```

Comment:
We may need to add more description to avoid possible misunderstandings: ` And If users don't set it, SDK will override it with "https://management.azure.com"`

Reply:
SDK will get it from the cloud setting. :)

Reply:
`And If users don't set it, SDK will get default value "https://management.azure.com" from cloud_setting to overwrite it`

Reply:
It is not necessarily to be "https://management.azure.com". If `base_url` is not set but `cloud_setting` is provided, we will use the value from cloud_setting.